proto/stream_service.proto

```diff
@@ -80,6 +81,9 @@ message BarrierCompleteResponse {
   uint32 worker_id = 5;
   map<uint32, hummock.TableWatermarks> table_watermarks = 6;
   repeated hummock.SstableInfo old_value_sstables = 7;
+  uint32 partial_graph_id = 8;
```

Comment:
What's the partial graph id referring to here?

Reply:
Ah I see, it's the id of the partial graph where the barrier is collected from.

Reply:
Barrier are injected and collected per partial graph. This refers to the partial graph that has finish collecting the barrier.